docs/rules/prefer-async-await.md

```diff
@@ -0,0 +1,50 @@
+# Prefer `async`/`await` over `.then()` (prefer-async-await)
+
+With the addition of `async`/`await` in ES2017, one can write asynchronous code
+in a synchronous-like manner.
```

Comment:
I think this sentence could be improved, although I'm not immediately sure how. But "synchronous-like" is not a term I've seen commonly used to describe this pattern... Let's iterate on it? Thanks!

Reply:
Maybe `synchronous-looking` would be a better fit? :thinking: